mobile/Features/Feed/Screens/PersonalInfo.jsx

```diff
@@ -0,0 +1,223 @@
+import {
+    Text, View, TextInput, TouchableOpacity, StyleSheet,
+  } from 'react-native';
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { DropdownSelect } from 'react-native-input-select';
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+        backgroundColor: '#E3F5F4',
+    },
+    arrowContainer: {
+        position: 'absolute',
+        top: 100,
+        left: 30,
+        flexDirection: 'row',
+        alignItems: 'center',
+    },
+    title: {
+        fontSize: 40,
+    },
+    inputContainer: {
+        marginTop: 20,
+        alignItems: 'flex-start',
+    },
+    inputWrapper: {
+        flexDirection: 'row', 
+        alignItems: 'center',
+        height: 63,
+        width: 350,
+        borderColor: 'gray',
+        borderWidth: 1,
+        marginTop: 5,
```

Comment:
i'm also seeing a lot of duplicate code for the styling 😳
if we move it to a separate file, having a single styling file and keeping the styling consistent across files would be pretty cool